crates/contract/src/call.rs

```diff
@@ -135,6 +139,27 @@ impl<T, P, D, N: Network> CallBuilder<T, P, D, N> {
     pub fn into_transaction_request(self) -> N::TransactionRequest {
         self.request
     }
+
+    /// Build a raw unsigned transaction for the call.
+    pub fn build_unsigned_raw_transaction(self) -> Result<Vec<u8>, TransactionBuilderError<N>>
+    where
+        N::UnsignedTx: SignableTransaction<Signature>,
+    {
+        let tx = self.request.build_unsigned().map_err(|e| e.error)?;
+        Ok(tx.encoded_for_signing())
+    }
+
+    /// Build a signed raw transaction for the call.
```

Comment:
these need a few more docs, not obvious what these are, what these return and when to use which